jetstream/workflows/run.yaml

```diff
@@ -48,6 +48,11 @@ spec:
         "--dataset_id={{workflow.parameters.dataset_id}}", 
         "--project_id={{workflow.parameters.project_id}}"
       ]
+      resources:
+        requests:
+          memory: 10Gi   # make sure there is at least 10Gb of memory available for the task
```

Comment:
Most tasks only use 1-2GB of memory, but occasionally the might be tasks that require more. Specifying this here, allows Kubernetes to assign jobs to different nodes (we have up to 3 with autoscaling) to ensure jobs have enough memory available. 

Before, all jobs got assigned to the same node since each job was configured to request 0GB of memory (which is the default if nothing is specified).